docs/GettingStarted.md

```diff
@@ -60,6 +61,15 @@ You are also welcome to download the source code and build the generator yoursel
 
 Datahelix is under active development so expect new features and bug fixes. Please feel free to share any issues, feature requests, or ideas via the [GitHub issues page](https://github.com/finos/datahelix/issues).
 
+### Install via apt get
```

Comment:
'apt get' or 'apt-get' whichever one is correct, let's use it consistently. Above it is 'apt-get'.